packages/driver/src/conUtils.ts

```diff
@@ -632,7 +632,9 @@ async function parseConnectDsnAndArgs(
     } else {
       throw new errors.ClientConnectionError(
         "Found 'edgedb.toml' but the project is not initialized. " +
-          "Run `edgedb project init`."
+          "If you are developing, run `edgedb project init`." + 
+          "If you are running this project in production, use a DSN." +
+          "Read more here: [https://github.com/edgedb/edgedb-js#configuring-the-connection](https://github.com/edgedb/edgedb-js#configuring-the-connection)"
```

Comment:
We're missing some whitespace here. Feel free to reformat this with string literals and have it all on one line.

Also just put the URL directly in since few, if any, of the places where you would see this error render Markdown.

Reply:
Okay, great catch! Thank you.